styles/variables/_colors.scss

```diff
@@ -0,0 +1,39 @@
+// Greys (layout colors)
+// These may be used for non-textual styling of components and elements
+$white: #fff !default;
+
+// Interim color variable for migration purposes.
+// Used as very-subtle background color for form fields. $grey-1 is too dark.
+$grey-0: #fafafa;
+
+$grey-1: #f2f2f2;
+$grey-2: #ececec;
+
+$grey-3: #dbdbdb;
+
+$grey-4: #a6a6a6;
+
+// Interim color variable for migration purposes, as the step between `$grey-4`
+// and `$grey-5` is large. Represents `base-semi` in proposed future palette,
+// minus blue tint.
+$grey-semi: #9c9c9c;
+
+// This is the lightest grey admissible on a white, $grey-0 or $grey-1
+// background to meet WCAG-AA text contrast requirements.
+$grey-5: #737373;
+
+// Interim color variable for migration purposes, as the step between `$grey-5`
+// and `$grey-6` is large. Represents `base-mid` in proposed future palette,
+// minus blue tint.
+// This is the lightest grey admissible on $grey-2, $grey-3
+$grey-mid: #595959;
+
+$grey-6: #3f3f3f;
+
+$grey-7: #202020;
+
+$brand: #bd1c2b;
+
+$link-hover: #84141e !default;
+
+$focus-outline: #51a7e8;
```

Comment:
I'm not sure how or why this maps to `color-focus-outline` in focus.scss, but it is. Can you help my understand this?

Reply:
Yes, in `styles/variables/_index.scss`:

https://github.com/hypothesis/frontend-shared/pull/22/files#diff-8c0e6fb927384efdc1f1df14adbd689d15f2836854e31a320217083e0f224257R1

`@forward 'colors' as color-*;`

It prefixes all of the members coming from the `_colors.scss` partial with the string `color-`

Reply:
I see, as we do in the client. Thanks.